rpm/kubelet.spec

```diff
@@ -38,7 +38,7 @@ Source5: https://dl.k8s.io/network-plugins/cni-plugins-%{ARCH}-v%{CNI_VERSION}.t
 
 BuildRequires: curl
 Requires: iptables >= 1.4.21
-Requires: kubernetes-cni = %{CNI_VERSION}
+Requires: kubernetes-cni = %{CNI_VERSION}-00
```

Comment:
Is this required and shouldn't this be `.0` or something?
I don't think RPMs use the same conventions as debs in this case

Reply:
@luxas Thanks for pointing out. I forgot to validate on Redhat.

Will have a try.

Reply:
@luxas RPMs use `-0` as postfix, such as `0.5.1-0` and `0.6.0-0`.
Updated. PTAL.